bin/build_rust

```diff
@@ -33,6 +33,11 @@ cd $top_dir/perf/smallbank_workload
 mkdir -p $top_dir/perf/smallbank_workload/bin
 cargo build && cp ./target/debug/smallbank-workload $top_dir/perf/smallbank_workload/bin/smallbank-workload
 
+echo -e "\033[0;32m--- Building intkey_workload ---\n\033[0m"
+cd $top_dir/perf/intkey_workload
+mkdir -p $top_dir/perf/intkey_workload/bin
+cargo build && cp ./target/debug/intkey-workload $top_dir/perf/intkey_workload/bin/intkey-workload
```

Comment:
`--release` ?

Reply:
Probably. It will make the builds longer. We will just have to be Ok with that.